src/Umbraco.Web/Models/Mapping/ContentModelMapper.cs

```diff
@@ -142,9 +142,33 @@ private static void AfterMap(IContent content, ContentItemDisplay display, IData
                 TabsAndPropertiesResolver.AddListView(display, "content", dataTypeService);
             }
 
+            var currentDocumentType = contentTypeService.GetContentType(display.ContentTypeAlias);

```

Comment:
For performance sake, this doesn't need to be looked up if it's not going to be used anyways (i.e. the user doesn't have permissions or we are not in debug mode.